### PR TITLE
add top1 and top5 accuracy

### PR DESCRIPTION
**Pull request by QiJune**
This PR also sort the OPs in tensor_ops.go with alphabetical order.

### CODE REVIEW

File: tensor.go [hunk 1]
```diff
@@ -385,6 +413,11 @@ func Eq(a, other Tensor) Tensor {
 	return Tensor{(*unsafe.Pointer)(&t)}
 }
 
+// Eq torch.eq
+func (a Tensor) Eq(other Tensor) Tensor {
```
Comment (wangkuiyi) [inline tensor.go:417]: What is the difference between Eq and Equal?

Comment (QiJune) [reply tensor.go:417]: [Eq](https://pytorch.org/docs/stable/generated/torch.eq.html) computes element-wise equality, the return value is a bool Tensor.

[Equal])(https://pytorch.org/docs/stable/generated/torch.equal.html) compares two tensors, the return value is a bool value.

File: tensor.go [hunk 2]
```diff
@@ -112,6 +100,22 @@ func (a Tensor) To(device Device, dtype int8) Tensor {
 	return Tensor{(*unsafe.Pointer)(&t)}
 }
 
+// CastTo cast tensor dtype
+func (a Tensor) CastTo(dtype int8) Tensor {
```
Comment (Yancey1989) [inline tensor.go:104]: Pytorch using `tensor.float()`, `tensor.int()` to cast to a type, do you want to follow this design pattern and provides `Float()`, `Int()` method to cast the Tensor?

Comment (QiJune) [reply tensor.go:104]: self.float() is equivalent to self.to(torch.float32).  Go does not support default parameters, `CastTo`is actually equal to `To(device=None)`. 

I will add `Float()` `Int()` if users needed.

Comment (Yancey1989) [reply tensor.go:104]: So do we need to provide `Float()` `Int()` method instead of `CastTo`, this is more similar to Pytorch.

**Review by Yancey1989 — APPROVED** [review]
LGTM

File: tensor.go (hunk 2 above)
Comment (QiJune) [reply tensor.go:104]: I believe that we need to expose the `Float` method too. I will add it in the next PR.

**Pull request merged**